StuDash/settings.py

```diff
@@ -51,11 +51,12 @@
 ]
 
 ROOT_URLCONF = 'StuDash.urls'
+TEMPLATE_DIR= "/Users/orbarel/development/projects/StuDash/StuDash_home/templates"
```

Comment:
Your username shouldn't find itself in the code at the upstream. Please find the right path that will work for everybody who clones the repo.
Maybe use relative instead of absolute path